test/riskMetrics/DynamicRiskTest.m

```diff
@@ -0,0 +1,28 @@
+% Test units of the DynamicRisk function
+%
+% ---------------------------------------------------------------------
+%
+% Copyright (C) 2020 Nunzio Camerlingo
+%
+% This file is part of AGATA.
+%
+% ---------------------------------------------------------------------
+time = datetime(2000,1,1,0,0,0):minutes(5):datetime(2000,1,1,0,0,0)+minutes(60); % length = 11;
```

Comment:
Add the missing 

addpath(fullfile('..','..','src','riskMetrics'));

Otherwise TravisCI will fail